.github/workflows/test.yml

```diff
@@ -5,21 +5,24 @@ on:
   push:
     branches:
       - master
+      - fixes_*
```

Comment:
Dit is de github action voor de cookiecutter, niet voor de gegenereerde code. Hier hoeft dit toch niet?